src/lib/apis/functions/index.ts

```
@@ -66,6 +66,41 @@ export const getFunctions = async (token: string = '') => {
 	return res;
 };
 
+export const loadFunctionByUrl = async (token: string = '', url: string) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)